File: src/main/java/no/nav/service/pensjon/regler/to/IdentifiserRegelendringerResponse.java
package no.nav.service.pensjon.regler.to;

import java.util.Vector;

import no.nav.domain.pensjon.regler.Pakkseddel;
import no.nav.domain.pensjon.regler.beregning2011.Regelendring;

/**
 * Responseobjekt for BEF2001 Identifiser regelendringer.
 * 
 * @author and2812
 *
 */
public class IdentifiserRegelendringerResponse extends ServiceResponse {

	/**
	 * Generated serial version UID.
	 */

	/**
	 * Listen av regel- og satsendringer som tjenesten fant for tidsrommet og input den ble kalt med.
	 * Listen er ikke sortert på dato, men hver dato i listen er unik. 
	 */
	private Vector<Regelendring> regelendringListe = new Vector<Regelendring>();
	/**
	 * Pakkseddel som indikerer om det oppstod feil ved tjenestekallet.
	 */
	private Pakkseddel pakkseddel;

	
}

File: src/main/kotlin/no/nav/pensjon/regler/to/IdentifiserRegelendringerResponse.kt
package no.nav.pensjon.regler.to


import no.nav.pensjon.regler.domain.beregning2011.Regelendring
import java.util.*

/**
 * Responseobjekt for BEF2001 Identifiser regelendringer.
 */
class IdentifiserRegelendringerResponse : ServiceResponse() {
    /**
     * Listen av regel- og satsendringer som tjenesten fant for tidsrommet og input den ble kalt med.
     * Listen er ikke sortert på dato, men hver dato i listen er unik.
     */
    var regelendringListe = Vector<Regelendring>()
}